content/test/unit/service/validations/Validations.spec.ts

```diff
@@ -13,6 +15,44 @@ import {
 } from '../../../../src/service/validations/Validator'
 import { MockedAccessChecker } from '../../../helpers/service/access/MockedAccessChecker'
 
+const avatarInfo = {
+  bodyShape: 'urn:decentraland:off-chain:base-avatars:BaseMale',
+  snapshots: {
+    face: 'https://peer.decentraland.org/content/contents/QmZdwrWnF2kLghFJ9kSj2brFEmywfAiqssr2LCqFj9HVWi',
```

Comment:
nit: those are hashes not links in the content server

Reply:
Should we validate that on the schema also? 🤔

Reply:
ok, that's good to know, we should!

Reply:
it seems too restrictive to use a regex as schema validation for these `hashes`, since it can be ipfs v1 or v2 (different formats) or a different algorithm in the future.

do you think we should add another validation method for that?

Reply:
That's true, but in v4 it must use IPFS. So, maybe we could implement the validation to be similar to what we already have on the content server:

```
  static isIPFSHash(hash: string) {
    return hash.startsWith('bafy') && hash.length === 59
  }
```

Reply:
once [this PR](https://github.com/decentraland/common-schemas/pull/20) is merged, will update this function to use the schema validation to avoid duplicated logic

Reply:
nicee